Userland/Libraries/LibWeb/HTML/History.h

```diff
@@ -25,7 +25,10 @@ class History final : public Bindings::PlatformObject {
     WebIDL::ExceptionOr<void> go(long delta);
     WebIDL::ExceptionOr<void> back();
     WebIDL::ExceptionOr<void> forward();
-    WebIDL::ExceptionOr<u64> length() const;
+    WebIDL::ExceptionOr<size_t> length() const;
+
+    size_t m_index { 0 };
```

Comment:
can we go for u64 or u32 here explicitly? there was a great purge of size_t from the code some time ago, be a shame to keep adding more uses.

Reply:
sure, fixed!